composer.json

```diff
@@ -103,7 +103,7 @@
         }
     },
     "scripts": {
-        "post-install-cmd": [
```

Comment:
remove this part , according to this issue 
[refactor composer.json - post-create-project-cmd](https://github.com/dotkernel/frontend/issues/531)